mne/io/edf/edf.py

```diff
@@ -438,6 +436,9 @@ def _get_info(fname, stim_channel, annot, annotmap, eog, misc, exclude,
     tal_ch_name = 'EDF Annotations'
     tal_chs = np.where(np.array(ch_names) == tal_ch_name)[0]
     if len(tal_chs) > 0:
+        logger.info('EDF annotations detected (consider using '
+                    'raw.find_edf_events() to extract events from these '
+                    'annotations)')
```

Comment:
maybe just "to extract them", to help distinguish them from proper MNE events (even though the name of the function makes it seem like they will be...)